app/javascript/app/components/table/table-component.jsx

```diff
@@ -11,192 +11,203 @@ import styles from './table-styles.scss';
 import { deburrCapitalize } from '../../utils/utils';
 import { getTableWidth, getResponsiveWidth } from './table-utils';
 
-class SimpleTable extends PureComponent {
-  rowClassName = ({ index }) => {
-    const { theme } = this.props;
+function TableComponent(props) {
+  const {
+    data,
+    hasColumnSelect,
+    activeColumns,
+    columnsOptions,
+    handleColumnChange,
+    setRowsHeight,
+    setColumnWidth,
+    headerHeight,
+    sortBy,
+    sortDirection,
+    handleSortChange,
+    ellipsisColumns,
+    setOptionsOpen,
+    setOptionsClose,
+    toggleOptionsOpen,
+    optionsOpen,
+    horizontalScroll,
+    firstColumnHeaders,
+    flexGrow,
+    splittedColumns,
+    theme,
+    titleLinks
+  } = props;
+  const rightTable = useRef();
+  useEffect(() => {
+    rightTable.current.scrollLeft = rightTable.current.scrollWidth;
+  }, rightTable.current);
```

Comment:
Hey, there is a warning in the console:
![image](https://user-images.githubusercontent.com/15097138/85843757-f0746900-b7a1-11ea-8ece-1fb97caf8f05.png)
shouldn't the dependency prop be inside an array?
```suggestion
  }, [rightTable.current]);
```